src/utils/monaco/yqlSuggestions/generateSuggestions.ts

```diff
@@ -15,6 +15,39 @@ import {
     WindowFunctions,
 } from './constants';
 
+const CompletionItemKind: {
```

Comment:
Why did you replace `monaco.languages.CompletionItemKind` with this object?

Reply:
to remove dependency on `monaco-editor` and depend only on types.